core/Pos/Binary/Core/Address.hs

```diff
@@ -118,6 +117,12 @@ For address there are two attributes:
 -}
 
 instance Bi (Attributes AddrAttributes) where
+    -- FIXME it was observed that for a 150kb block, this call to
```

Comment:
WIP?

Reply:
I still do not understand whether it's a thing to be done is this PR or not. Is this comment relevant at all?